WBooks/HomeViewController.swift

```diff
@@ -42,21 +58,19 @@ final class HomeViewController: UITabBarController {
     
     private func setupTabBarItems() {
         var tabBarControllers = [UIViewController]()
+        let booksRepository = NetworkingBootstrapper.shared.createBooksRepository()
         
-        let getBooks: () -> SignalProducer<[Book], NSError> = {
-            return SignalProducer(value: [Book(id: 0, author: "First author and a name that is so big", title: "First Title", imageUrl: "http:wolox-training.s3.amazonaws.com/uploads/content.jpeg"),
-                                          Book(id: 0, author: "Second author", title: "Second title and not an URL image", imageUrl: "this is not an URL"),
-                                          Book(id: 0, author: "Third author", title: "Third title", imageUrl: "http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg"),
-                                          Book(id: 0, author: "Large title and author and failed image", title: "Large title and author and failed image", imageUrl: "http://failedimage.jpg")])
-        }
-        let bookViewModel = BooksViewModel(getBooks: getBooks)
-        let libraryViewController = BooksTableViewController(viewModel: bookViewModel)
-        libraryViewController.title = Constants.libraryName
+        let libraryNavigationController = UINavigationController()
+        let libraryViewModel = LibraryViewModel(libraryRepository: booksRepository)
+        let libraryViewController = LibraryViewController(libraryViewModel: libraryViewModel) 
+        libraryViewController.title = Constants.libraryName.uppercased()
+        setupNavigationController(viewContoller: libraryViewController)
         libraryViewController.tabBarItem = UITabBarItem(title: Constants.libraryName,
                                                         image: Constants.libraryImage,
                                                         selectedImage: Constants.librarySelectedImage)
+        libraryNavigationController.viewControllers = [libraryViewController]
```

Comment:
I think there is a rootViewController or something like that to set it. If not, I think you can create the navigation controller with the root controller.